install_sct

```diff
@@ -91,8 +93,12 @@ for installation instructions check: https://git-scm.com/book/en/v2/Getting-Star
 
   if [[ ! $(which gcc) ]]; then
     if [[ $OS == "osx" ]]; then
-      read -p 'You need to install gcc before you can use the SPINALCORDTOOLBOX. Do you want to install gcc now? [y]es/[n]o: ' -r GCC_INSTALL
+      echo -e "\nYou need to install gcc before you can use the SPINALCORDTOOLBOX."
+      read -p 'Do you want to install gcc now (accepting to install gcc will also install brew in case it is not installed already)? [y]es/[n]o: ' -r GCC_INSTALL
       if [[ $GCC_INSTALL =~ [Yy](es)? ]]; then
+        if [[ ! $(which brew) ]]; then
+          /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+        fi
         brew install gcc
```

Comment:
i would install gcc and brew it in silence mode (in case installer asks for confirmation, etc. that the user might not be confortable with)

Reply:
Sure. Changed :+1